README.md

```diff
@@ -91,23 +109,7 @@ Then, run the following command:
 $ pod install
 ```
 
-### Swift Package
-The [Swift Package Manager](https://swift.org/package-manager/ "") is a tool for automating the distribution of Swift code and is integrated into the swift compiler.
-
-Once you have your Swift package set up, adding Alamofire as a dependency is as easy as adding it to the `dependencies` value of your `Package.swift`.
-```swift
-dependencies: [
-    .package(url: "https://github.com/web3swift-team/web3swift.git", .upToNextMajor(from: "3.0.0"))
-]
-```
-
-## Example usage
-In the imports section:
-
-```swift
-import web3swift
-import Web3Core
-```
+> **WARNING**: CocoaPods is a powerful tool for managing dependencies in iOS development, but it also has some limitations that need to be considered such as its limited control over dependencies, lack of transparency, complex setup and slow performance.
```

Comment:
I'd also add:
> We highly recommend using SPM first as using CocoaPods will delay new updates and bug fixes being delivered to you.

The rest is good.
@yaroslavyaroslav What's your opinion on this? You have done more work with CocoaPods.

Reply:
Thanks, I have added the additional sentence to the warning.